pyproject.toml

```diff
@@ -52,7 +52,7 @@ main = "kinto:main"
 include = ["kinto*"]
 
 [tool.setuptools.package-data]
-"*" = ["*.tpl", "*.sql", "kinto/plugins/admin/build/*", "VERSION"]
+"*" = ["*.tpl", "*.sql", "*.html", "*.css", "*.js", "*.png", "VERSION"]
```

Comment:
> `kinto/plugins/admin/build/*`

Did this only include `index.html`? Would a glob of `kinto/plugins/admin/build/**/*` work?

Reply:
Found the pattern that works best I think:
`plugins/admin/build/**`